app/api/v2/sessions.py

```diff
@@ -0,0 +1,73 @@
+""" bancho.py's v2 api's authorization """
+from __future__ import annotations
+
+import uuid
+
+from fastapi import APIRouter
+from fastapi import Depends
+from fastapi.security import HTTPAuthorizationCredentials as HTTPCredentials
+from fastapi.security import HTTPBearer
+
+from app.api.v2.common import responses
+from app.api.v2.common.responses import Success
+from app.api.v2.models.sessions import LoginForm
+from app.api.v2.models.sessions import Session
+from app.api.v2.models.sessions import SessionUpdate
+from app.repositories import sessions as sessions_repo
+from app.usecases import sessions
+
+router = APIRouter()
+oauth2_scheme = HTTPBearer()
+
+
+@router.post("/sessions")
+async def create_session(args: LoginForm) -> Success[Session]:
+    data = await sessions.authorize(username=args.username, password=args.password)
+    if data is None:
+        return responses.failure(message="Failed to create session.")
+
+    response = Session.from_mapping(data)
+    return responses.success(response)
```

Comment:
i think we want to implement oauth with client id and secret rather than using name & password like this

see osu api v2 for example